web/assets/js/GoogleAnalyticsService.js

```diff
@@ -0,0 +1,77 @@
+class GoogleAnalyticsService {
+
+    constructor() {
+        this.gaDataAttr = 'data-ga-action';
+        this.gaCategoryAttr = 'data-ga-category';
+        this.gaLabelAttr = 'data-ga-label';
+    }
+
+    init() {
```

Comment:
CS: By default `init` should be the last defined function in a class. That's how we do it.